charts/canary/templates/rbac/role.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Let's change this to a `Role` and limit it to `namespace: {{ .Release.Namespace }}`